src/client/datascience/types.ts

```diff
@@ -863,22 +863,25 @@ export interface IJupyterExtraSettings extends IJupyterSettings {
     };
 }
 
-// Get variables from the currently running active Jupyter server
+// Get variables from the currently running active Jupyter server or debugger
 // Note: This definition is used implicitly by getJupyterVariableValue.py file
 // Changes here may need to be reflected there as well
 export interface IJupyterVariable {
     name: string;
     value: string | undefined;
     executionCount?: number;
     supportsDataExplorer: boolean;
+    supportsSlicing?: boolean;
     type: string;
     size: number;
     shape: string;
+    dataDimensionality?: number;
     count: number;
     truncated: boolean;
     columns?: { key: string; type: string }[];
     rowCount?: number;
     indexColumn?: string;
+    maximumRowChunkSize?: number;
```

Comment:
This is set in IJupyterVariables from DebuggerVariables to tell the UI the maximum chunk size it can ask for at once.